ClimsoftVer4/ClimsoftVer4/formClimat.vb

```diff
@@ -1,3 +1,19 @@
-﻿Public Class formClimat
+﻿'CLIMSOFT - Climate Database Management System
+' Copyright (C) 2017
+'
+' This program is free software: you can redistribute it and/or modify
+' it under the terms of the GNU General Public License as published by
+' the Free Software Foundation, either version 3 of the License, or
+' (at your option) any later version.
+'
+' This program is distributed in the hope that it will be useful,
+' but WITHOUT ANY WARRANTY; without even the implied warranty of
+' MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+' GNU General Public License for more details.
+'
+' You should have received a copy of the GNU General Public License
+' along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+Public Class formClimat
 
 End Class
```

Comment:
The red symbol here (after "End Class") means that there is no line break (e.g. an empty line) at the end of the file.  Some systems historically need this, so it is considered good practice in all code files to always end every file with exactly one newline. We could do this at the same time as updating the licence info, but it is not essential